lib/regaliator/version.rb

```diff
@@ -1,3 +1,3 @@
 module Regaliator
-  VERSION = '4.0.1'
+  VERSION = '4.0.2'
```

Comment:
Please use the version `4.1.0`, if we follow the semantic versioning we are adding a feature (a new API version) so it's a minor upgrade and not a patch.

http://semver.org/